plugins/network/NetEng.hlf

```diff
@@ -44,30 +44,34 @@ $ #Network browser configuration#
  Enables adding "Network" item to the disks menu.
 
  #Add to plugins menu#
- Enables adding "Network" item to the plugins menu.
+ Enables adding "Network browser" item to the plugins menu.
```

Comment:
The English hlf is the baseline for all the other translations.
It's easier to spot the untranslated bits by comparing with it.
If you change anything in it, please update all the other hlfs as well, it's a pain when they go out of sync.

Reply:
Please add the "Scan network" topic to help file - now it is not described.

Reply:
@BestiaPL  in the next patch